src/services/conseillers/conseillers.class.js

```diff
@@ -1103,46 +1126,60 @@ exports.Conseillers = class Conseillers extends Service {
         }).toJSON());
         return;
       }
-      if (String(conseiller._id) !== String(user.entity.oid)) {
-        res.status(403).send(new Forbidden('User not authorized', {
-          userId: userId
-        }).toJSON());
-        return;
-      }
       if (!conseiller?.mailAModifier && !conseiller?.mailProAModifier) {
         res.status(404).send(new NotFound('mailAModifier not found').toJSON());
         return;
       }
       if (existTokenMail) {
         try {
-          await this.patch(conseiller._id, {
-            $set: { email: conseiller.mailAModifier },
+          await pool.query(`UPDATE djapp_coach
+          SET email = $2
+              WHERE email = $1`,
+          [conseiller.email, conseiller.mailAModifier.toLowerCase()]);
```

Comment:
A vérifier pas besoin de :

```suggestion
          SET email = lower($2)
              WHERE lower(email) = lower($1)`,
          [conseiller.email, conseiller.mailAModifier.toLowerCase()]);
```

Reply:
Bien vu, ça réduit d'avantage la casse !